src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -93,10 +106,10 @@ export class BaseEditCourseForm extends React.Component {
     return (
       <div className="edit-course-form">
         <form onSubmit={handleSubmit} id={id}>
+          <FieldLabel text="Course" className="mt-4 mb-2" />
```

Comment:
Is this Dillon UI or Carol UI? It's good either way, just curious.

Reply:
This is Dillon UI that was somewhat discussed over slack.